core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -66,6 +66,10 @@ oppia.controller('FeedbackTab', [
     $q.all([userInfoPromise, threadPromise]).then(function() {
       $rootScope.loadingMessage = '';
     });
+    $scope.fetchAgain = function() {
```

Comment:
Change` fetchAgain` to `fetchThreadsAgain`. It is more meaningful. :)

Reply:
Done